src/components/testbutton/types.ts

```
@@ -0,0 +1,7 @@
+export interface TestButtonProps {
+  // Add component-specific props here
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)